TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/FeatureNames.cs

```diff
@@ -8,4 +8,5 @@ public static class FeatureNames
     public const string RoutesToProfessionalStatus = "RoutesToProfessionalStatus";
     public const string NewUserRoles = "NewUserRoles";
     public const string DqtNotes = "DqtNotes";
+    public const string NewPersonDetails = "NewPersonDetails";
```

Comment:
We could have a single feature that represents when the contact record is migrated to TRS; perhaps `PersonMigration`?

Reply:
Changed to `ContactsMigrated`